src/parser.js

```diff
@@ -36,11 +36,6 @@ export default class Parser extends EventEmitter {
   constructor(options = {}) {
     super(options)
     this.options = options
-
-    if (options.reactNamespace) {
-      lexers.js = lexers.jsx
```

Comment:
This line was apparently shady, because it modified the actual `lexers` constant instead of field on an instance. Because of that a test provided a false positive [here](https://github.com/i18next/i18next-parser/pull/235/files#diff-7bf59ee18e32a02823031db5b25b384cR1447), because there was no reactNamespace the test should've failed, but it passed because a test before it overwrote the constant.